analysis/tests/src/Destructuring.res

```diff
@@ -0,0 +1,18 @@
+type x = {name: string, age: int}
+
+let x = {name: "123", age: 12}
+
+let {name, } = x
+//         ^com
+
+// let {} = x
+//      ^com
+
+let f = (x: x) => {
+  let {name, } = x
+  //         ^com
+  name
+}
+
+// let f = (x: x) => { let {} = x }
```

Comment:
Probably better to put just the destructure inside of a comment and in the uncommented fn body above - a full function without compilation won't work even after we fix the underlying issues here.

Reply:
Better now?

Reply:
Yes, great.